.flake8

```diff
@@ -4,7 +4,7 @@ ignore =
   F401  # unused import, defer to pylint
   W503  # allow line breaks before binary ops
   W504  # allow line breaks after binary ops
-  E203  # allow whitespace before ':' (https://github.com/psf/black#slices) 
+  E203  # allow whitespace before ':' (https://github.com/psf/black#slices)
```

Comment:
Was adding this whitespace intentional?

Reply:
No, I'm actually removing the whitespace herebecause trailing whitespace is considered poor formatting.

Reply:
Sorry, remove, I just meant to not format areas you did not touch.